Content.Server/Sound/Components/SpamEmitSoundRequirePowerComponent.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Server.Sound.Components;
+
+/// <summary>
+/// Enables or disables an SpamEmitSound component depending
+/// on the powered state of the entity.
+/// </summary>
+[RegisterComponent]
+public sealed partial class SpamEmitSoundRequirePowerComponent : Component
+{
+}
```

Comment:
Needs predicting.

Reply:
I'll work on it, but with PowerChangedEvent and PowerNetBatterySupplyEvent only raised on the server, and SpamEmitSoundComponent running serverside, I don't think it'll make much of a difference.